src/services/requestHandler.js

```diff
@@ -10,8 +10,15 @@ const axiosInstance = () => {
   if (process.env.NODE_ENV === 'development') {
     apiUrl = 'http://localhost:3000/api';
   }
+
+  let token = '';
+  if (localStorage.getItem('token')) {
+    token = localStorage.getItem('token');
+    token = JSON.parse(token);
```

Comment:
You don't need to parse the token, it is already a string

Reply:
Ok!